pkg/broker/broker.go

```diff
@@ -194,33 +194,35 @@ func (a AnsibleBroker) Bootstrap() (*BootstrapResponse, error) {
 	var specs []*bundle.Spec
 	var imageCount int
 
-	// Remove all non apb-push sourced specs that have been saved.
-	pushedSpecs := []*bundle.Spec{}
```

Comment:
@dymurray Can we make a follow up issue to delete the apb endpoints? Can we offically deprecate and delete them as we move to the new CLI?

Reply:
+1